src/SFA.DAS.EmployerAccounts/Interfaces/ICacheStorageService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerAccounts.Interfaces
+{
+    public interface ICacheStorageService
+    {
+        Task<T> RetrieveFromCache<T>(string key);
```

Comment:
As this interface is to do with CacheStorage the method names don't need to specify that data is being retrieved form the cache.  Maybe just call the methods Get, Save and Delete to remove the tautology.

Reply:
Im just following the pattern that was used in the other repos.

Reply:
I don't mind this being changed to Get, Save and Delete.